src/models/Feed.ts

```diff
@@ -13,7 +13,7 @@ export interface FeedItem {
   itunes: {
     summary: string,
     explicit: string,
-    duration: string,
+    duration: number,
```

Comment:
```suggestion
    duration: string,
```
This `FeedItem` interface is what comes from the RSS, so everything there is string.